app/page.tsx

```diff
@@ -27,6 +28,12 @@ export default function Home() {
             <section>
                 <AboutTailspin />
             </section>
+            <section
+                id='stepper'
+                className='flex h-screen items-center justify-center'
```

Comment:
StepperCard is essentially TabSection. Should remove TabSection.

Reply:
done